### PR TITLE
Added support for android view intent action

### PR DESCRIPTION
**Pull request by odam1**

### CODE REVIEW

File: example/ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -717,7 +755,7 @@
 				CODE_SIGN_ENTITLEMENTS = Runner/Runner.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = BN2V9FW285;
```
Comment (GRBurst) [inline example/ios/Runner.xcodeproj/project.pbxproj:758]: I think this is set to "" on purpose ;-) I suggest leaving it out.
```suggestion
				DEVELOPMENT_TEAM = "";
```

File: example/ios/Runner.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -596,7 +634,7 @@
 				MARKETING_VERSION = 1.0.0;
 				MTL_FAST_MATH = YES;
 				OTHER_LDFLAGS = "";
-				PRODUCT_BUNDLE_IDENTIFIER = "com.kasem.sharing.Sharing-Extension";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.odamsoft.sharing.Sharing-Extension";
```
Comment (GRBurst) [inline example/ios/Runner.xcodeproj/project.pbxproj:637]: Keep it like `com.kasem....`
```suggestion
				PRODUCT_BUNDLE_IDENTIFIER = "com.kasem.sharing.Sharing-Extension";
```

**Review by GRBurst — CHANGES_REQUESTED** [review]

**Comment by GRBurst** [comment]
I added some representative change request, but there is more of your personal ids that need to be reverted ;-)

**Comment by filipenanclarez** [comment]
intent.action.VIEW is handled in package as url type. When user try open a file, android system send the uri of this file to intent.

if you need open file with dart:io functions like readAdBytes or readAsString, you should convert the Uri to full path. You can do this with [uri_to_file](https://pub.dev/packages/uri_to_file) package.

So, you can handle action.VIEW with this code in flutter side:

```
import 'package:uri_to_file/uri_to_file.dart';
...
// For sharing or opening urls/text coming from outside the app while the app is closed
ReceiveSharingIntent.getInitialText().then((String? value) async {
    String _sharedText = value ?? "";
    debugPrint('path of file: ${Uri.parse(_sharedText).path}');
    File file = await toFile(_sharedText);
    var fileContent = await file.readAsString();
    ...
});
```

**Pull request closed without merging**